src/main/default/classes/TriggerHandler.cls

```diff
@@ -41,7 +49,7 @@ public class TriggerHandler {
      *   Invokes correct handlers as per the context of Trigger and available registered handlers
      */
     public void manage() {
-        if (objectDisabled(sObjectType))
+        if (!triggerConfig.enabled)
```

Comment:
seems like this will return an NPE if there is no config setup

Reply:
personally i'd vote for ensuring TriggerConfigService.getConfig has a default result and never returns null, so downstream code doesn't have to worry about this sort of NPE

Reply:
that would actually require looking at the statically bound trigger handlers and creating `HandlerConfig` for each

Reply:
or maybe not... just create a default `ObjectConfig` with an empty `handler` map.

Reply:
fwiw, i've got some generic handlers i've used that have multiple instances that are bound with different constructors, don't really think the framework should be responsible for catching an issue where someone directly binds a class and injects the same handler ...

quick aside, and at least for now, direct bindings are required if you want to have constructor params (rare, but still really useful)

Reply:
For now adding null checks and also added some checks to only allow one run of each handler per event